src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -4022,6 +4022,16 @@
      *         |
      *         +--COMMENT_CONTENT
      * </pre>
+     *
+     * <p>For example:</p>
+     * <pre>
+     * // Comment content
+     * </pre>
+     * <p>parses as:</p>
+     * <pre>
+     * --SINGLE_LINE_COMMENT -&gt; //
```

Comment:
Remove leading "--" to indent whole code to right

Reply:
On it

Reply:
Done, Please check it again.